cms/models/pluginmodel.py

```diff
@@ -250,6 +250,23 @@ def get_plugin_instance(self, admin=None):
         self._inst = instance
         return self._inst, plugin
 
+    def get_bound_plugin(self, admin=None):
```

Comment:
I don't want get_bound_plugin() to be related to get_plugin_instance() or the concept of plugin admin classes in any way.

The goal is to remove get_plugin_instance() at some point.